### DISCUSSION TITLE
Configuring reverse proxy with nginx-proxy-manager

### DISCUSSION THREAD

**Discussion by jtermi03** [sct/overseerr#959] [Help] [answered]
Hi, I'm having a little trouble figuring out the right way to configure nginx proxy manager to get from my newly registered domain to overseerr and am hoping someone can help me bridge the gap. I have some Linux server experience but none with this sort of stuff. I followed [this guide](https://smarthomepursuits.com/how-to-install-nginx-proxy-manager-in-docker) 
and am able to access the default 

> "Congratulations! You've successfully started the Nginx Proxy Manager. If you're seeing this site then you're trying to access a host that isn't set up yet."

 message, so I assume my problem is between Nginx Proxy Manager and Overseerr (which is functional). 

My suspicion is that my lack of a config file is my problem. I saw [this nginx example](https://docs.overseerr.dev/extending-overseerr/reverse-proxy-examples) on the Overseerr documentation, but I can't connect the dots on how to get that into NPM. I tried pasting it into the config section of the Advanced tab of my proxy host settings, but that knocked it offline. 

![image](https://user-images.githubusercontent.com/32025689/108259706-6d889000-712f-11eb-8751-2f5f416aa89d.png)

**Comment by TheCatLady** [comment, marked as answer]
Hi again, @jtermi03!  While we provide sample reverse proxy configurations in our documentation, we do not provide support for Nginx Proxy Manager specifically.  If you need help with Nginx Proxy Manager, your best bet is through their support channels.

That said, we also do have an active [Discord server](https://discord.gg/PkCWJSeCk7).  You can try asking in `#off-topic` to see if anyone else has any experience using Nginx Proxy Manager.

And if you're willing to consider other options, my personal favorite reverse proxy solution is [SWAG](https://github.com/linuxserver/docker-swag).  It comes pre-loaded with sample proxy configurations for various services, including Overseerr.

**Reply by jtermi03** [reply]
Ah, understandable! Well, the good news is that it was all due to me not understanding how subdomains and redirects work, so things are looking up!

**Comment by nathan5752** [comment]
You need to use http (not https) for the "scheme".  You can for SSL and use HTTP/2 Support.  For the SSL Certificate you need to use the certificate from cloudflare or other source if you have strict encryption enabled, otherwise choose request a new SSL.  You should not require any Custom Nginx Configuration
